phpstan-baseline.neon

```diff
@@ -135,3 +135,7 @@ parameters:
 			count: 1
 			path: tests/SnapshotTest.php
 
+		-
```

Comment:
Not sure if this is the right indentation level

Reply:
Thanks, I'll check

Reply:
hmm, basically I copied lines 28-31 and did not change indentation at all 🤔 

It's the same like in the whole file, four spaces everywhere..

Reply:
@thePanz anything I can do here?